external/netfx/netfx.depproj

```diff
@@ -21,27 +21,9 @@
       <RefPath Condition="'$(ForShims)' != 'true'">$(RefPath)</RefPath>
       <RefPath Condition="'$(ForShims)' == 'true'">$(NetFxRefPath)</RefPath>
     </BinplaceConfiguration>
-  </ItemGroup>
-
-  <!-- Filter the targeting pack to just these assemblies which we need netcoreapp shims for -->
-  <ItemGroup>
-    <TargetingPackReference Include="mscorlib" />
```

Comment:
We can filter what goes into netfx.  Previously I had a separate filter for netfx/net4* configured projects and shims.  This broke because they needed to use the same folder.  The shims will always be a smaller subset than the libs we build against so we need filtering in the shims.  We can easily add back filtering to the depprojs at will in the future.